Automattic/wp-calypso
# UpsellNudge: Add UpsellNudge block to unify upsell nudges going forward

#### Changes proposed in this Pull Request

* Add an `UpsellNudge` block to replace the use of `SidebarBanner` and `Notice` components in the sidebar.
* This is a wrapper for `Banner`, to restyle it to match the new nudges we're proposing for the sidebar. See #38733 for changes made to `Banner` to add support for a new compact style.
* Eventually we hope to consolidate all upsell nudges to use this block.

Example implementation:
<img width="289" alt="Screen Shot 2020-01-13 at 4 46 24 PM" src="https://user-images.githubusercontent.com/2124984/72294979-a6992b80-3624-11ea-8e76-5e573744b7c9.png">

#### Testing instructions

* Switch to this PR
* Check to make sure the new block does not break anything. It isn't implemented yet, but it should not break existing uses of `Banner`.
